Proposed rewrite to "Creating Microsoft Azure Virtual Machines for NGINX Open Source and F5 NGINX Plus" (#608)
nginx/documentation: content/nginx/deployment-guides/microsoft-azure/virtual-machines-for-nginx.md

   <a href="/nginx/images/azure-create-vm-basics.png"><img src="/nginx/images/azure-create-vm-basics.png" alt="screenshot of 'Basics' tab on Azure 'Create a virtual machine' page" width="1024" height="1168" class="aligncenter size-full wp-image-64995" style="border:2px solid #666666; padding:2px; margin:2px;" /></a>

   <span id="create-vm_Networking"></span>

> 5. For [Active-Active HA for NGINX Plus on Microsoft Azure Using the Azure Standard Load Balancer]({{< ref "high-availability-standard-load-balancer.md" >}}):

Reviewer: The original text of step 5 is still more understandable for me (although I understand it's not perfect) so I'd rather use the original version, thanks

 Two major virtual machines run NGINX Plus behind an Azure Load Balancer. And both VMs require **Standard** SKU public IP addresses. By default, Azure assigns **Basic** SKU IPs. So, you must manually change this during VM setup. 

To avoid errors, assign **Standard** public IP addresses to the VMs you’ll use in the deployment. 

> When this guide was first published, the hourly cost for the six VMs was only $0.008. And this costlier than VMs with basic IP addresses. For current pricing, see the [Microsoft documentation](https://azure.microsoft.com/en-us/pricing/details/ip-addresses/).

Reviewer 2: The first sentence does not make sense and needs to be rewritten.

Depending on your work load, you might need up to 6 VMs. Follow these steps:
  - Open the **Networking** tab on the **Create a virtual machine** window.

>   - Click  _Create new_ below the **Public IP** field.

Reviewer 2: ```suggestion
  - Select  _Create new_ below the **Public IP** field.
```

  - In the **Create public IP address column** that opens, click the **Standard Radio button** under **SKU**. 
  - In the **Name** field, accept the default created by Azure, ngx-plus-1-ip. 

>   - Click the _Ok_  button.

Reviewer 2: ```suggestion
  - Select _Ok_.
```
When referring to a button by the label it is showing on screen there is no need to add "button" after. "Select Apply, Select Cancel, etc"

{{< img src="nginx/images/azure-create-vm-networking.png" alt="The 'Networking' tab on Azure's 'Create a virtual machine' page" >}}

6. Here, you can select non-default values on the **Disks**, **Networking**, **Management**, **Advanced**, and **Tags** tabs. By default, Azure assigns **Premium SSD** for the OS disk on the disk tab, but you can choose a cheaper option like **Standard HDD**.

   After you've completed your changes, click the _Review + create_ button at the bottom of the **Create a virtual machine** page.

   You’ll find a summary of your setting under the **Validation passed** banner. Ensure they’re suitable. Then, click on <span style="background-color:#137ad1; color:white;"> Create </span> 

>    If you generated a new key pair in [Step 4](#create-vm_Basics), a _Generate new key pair_ window pops up. Click on the _Download key and create private resource_ button.

Reviewer 2: ```suggestion
   If you generated a new key pair in [Step 4](#create-vm_Basics), a _Generate new key pair_ window pops up. Select _Download key and create private resource_.
```

   <a href="/nginx/images/azure-create-image-create-vm.png"><img src="/nginx/images/azure-create-image-create-vm.png" alt="screenshot of details page for Azure 'ngx-plus-1-image' image" width="1024" height="426" class="aligncenter size-full wp-image-64986" style="border:2px solid #666666; padding:2px; margin:2px;" /></a>

> The **Create a VM** page looks like Step 4 in Creating a Microsoft Azure Virtual Machine, but some fields are pre-filled from your image. The **Image** field now shows the image name instead of an OS. Go back to that [step](#create-vm_Basics) to finish creating your VM.

Reviewer 2: ```suggestion
The **Create a VM** page is similar to the Step 4 in Creating a Microsoft Azure Virtual Machine, but some fields are pre-filled from your image. The **Image** field now shows the image name instead of an OS. Go back to that [step](#create-vm_Basics) to finish creating your VM.
```
